src/tests/functional/plugin/cpu/single_layer_tests/convolution.cpp

```diff
@@ -215,6 +216,20 @@ class ConvolutionLayerCPUTest : public testing::WithParamInterface<convLayerCPUT
     // now we have a vector of CPU params passed to the test and inference is made for each param in one test
     void infer() override {
         auto expectedOutputs = calculate_refs();
+        std::shared_ptr<ov::Node> convOpPtr;
+
+        auto ops = function->get_ops();
+        for (const auto &op : ops) {
+            if (std::string {op->get_type_name()} == "Convolution") {
+                convOpPtr = op;
+                break;
+            }
+        }
+        if (!convOpPtr)
+            IE_THROW() << "No operation with type Convolution in test function";
```

Comment:
NIT. Is not it better to use ASSERT_TRUE macro from gtest since this is actually a test?

Reply:
changed to `ASSERT_FALSE(convOpPtr == nullptr)`, thanks)